handlers/external/handler.go

```diff
@@ -366,7 +366,7 @@ func (h *handler) Send(ctx context.Context, msg courier.MsgOut, res *courier.Sen
 		}
 
 		if responseCheck != "" && !strings.Contains(string(respBody), responseCheck) {
-			return courier.ErrResponseUnexpected
+			return courier.ErrFailedWithReason("mt-response-check", fmt.Sprintf("response not containing '%s'", responseCheck))
```

Comment:
Hmm I think we want basically the same as `ErrResponseUnexpected` but not logged.. so maybe just a new error constant:

```go
var ErrResponseUnexpectedUnlogged error = &SendError{
	msg:       "response not expected values",
	retryable: false,
	loggable:  false,
	clogCode:  "response_unexpected",
	clogMsg:   "Response doesn't match expected values.",
}
```